pom.xml

```diff
@@ -82,62 +82,62 @@
         <buildnumber-maven-plugin.version>1.4</buildnumber-maven-plugin.version>
         <dependency-check-maven.version>6.1.6</dependency-check-maven.version>
         <exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
-        <jacoco-maven-plugin.version>0.8.6</jacoco-maven-plugin.version>
+        <jacoco-maven-plugin.version>0.8.7</jacoco-maven-plugin.version>
         <maven-antrun-plugin.version>3.0.0</maven-antrun-plugin.version>
         <maven-assembly-plugin.version>3.3.0</maven-assembly-plugin.version>
-        <maven-bundle-plugin.version>5.1.1</maven-bundle-plugin.version>
```

Comment:
COMN-280 indicates this should be updated to 5.1.3, but I assume 5.1.4 is intentional

Reply:
It is intentional. Version 5.1.4 was released after I’d written the ticket.